docs/_build/html/.buildinfo

```diff
@@ -0,0 +1,4 @@
+# Sphinx build info version 1
+# This file hashes the configuration used when building these files. When it is not found, a full rebuild will be done.
+config: bfa60070bd1108aa0d4716c7cdb93896
+tags: 645f666f9bcd5a90fca523b33c5a78b7
```

Comment:
The PR contains the sphinx output.

Reply:
I've removed in https://github.com/bbcmicrobit/micropython/pull/695/commits/e46dfcf594fa6b39b00e596363d817d631035508 and added `_build/*` to gitignore in https://github.com/bbcmicrobit/micropython/pull/695/commits/81f37d9e5e97093647a4c99948f53152927e9cd3